pbm/storage/s3/s3.go

```diff
@@ -118,6 +119,10 @@ func New(opts Conf, l *log.Event) (*S3, error) {
 const defaultPartSize = 10 * 1024 * 1024 // 10Mb
 
 func (s *S3) Save(name string, data io.Reader, sizeb int) error {
+	storageClass := s.opts.StorageClass
+	if storageClass == "" {
+		storageClass = s3.StorageClassStandard
+	}
```

Comment:
Although storage class is related only to upload (S3.Save method) I'd like to move this check to [the Cast() method](https://github.com/percona/percona-backup-mongodb/blob/bb6286a6cbeb749d32b62b29aca84b4da54f09d6/pbm/storage/s3/s3.go#L56). This method is called during config set so no need to run checks on each save. Besides, it'd be more sound with the rest of the code.

Reply:
Good suggestion  @dAdAbird, I'll change that.

Reply:
Fix in latest commit.